components/calendar/src/islamic.rs

```diff
@@ -481,19 +485,21 @@ impl DateTime<UmmalQura> {
         hour: u8,
         minute: u8,
         second: u8,
-    ) -> Result<DateTime<UmmalQura>, CalendarError> {
+    ) -> Result<DateTime<UmmAlQura>, CalendarError> {
         Ok(DateTime {
             date: Date::try_new_ummalqura_date(year, month, day)?,
             time: types::Time::try_new(hour, minute, second, 0)?,
         })
     }
 }
 
-impl UmmalQura {
-    /// Constructs a new UmmalQura Islamic Calendar
+impl UmmAlQura {
+    /// Constructs a new UmmAlQura Islamic Calendar
     pub fn new() -> Self {
         Self
     }
+
+    // Since March 2002, the month begins on the first evening after the conjuction on which the moon sets after the sun, this is the criterion expressing the rule.
```

Comment:
I would express this in concrete terms as the two things must hold:

 - The moon is a waxing crescent at sunset the previous day
 - The moon sets after the sun does that evening

and note that the Islamic day starts on the previous day's sunset